### PR TITLE
fix: highlighting is not correct between quote escape sequences

### PR DESCRIPTION
**Pull request by alacambre-yseop**
@vholten-yseop 
Should be better now.
Try it and tell me what do you think.

image

### CODE REVIEW

**Comment by alacambre-yseop** [comment]
To test, you just need to get in the Jenkins artefacts the `*.vsix` file, then install it in VSCode.

**Comment by vholten-yseop** [comment]
Thanks! Highlighting is consistent now. 
With my color scheme:
yml-extension-fixed

File: syntaxes/yml.tmLanguage.YAML-tmLanguage [hunk 1]
```diff
@@ -200,21 +153,38 @@ repository:
...
+        begin: >
+            (?x)
+            # Keywords that have arguments and may be wrongfully found as functions.
+            (?!\\?(case|for|foreach|if|ifExpr|switch|switchExpr|switchExprExclusive|while))
+            (\\?[\p{L}_\p{N}]+(::?[\p{L}_\p{N}]+)*)
+            \s*
+            (\()
+        beginCaptures:
+            '2': { name: entity.name.function.yseop }
+            # '3': { name: entity.name.function.yseop }
```
Comment (vholten-yseop) [inline syntaxes/yml.tmLanguage.YAML-tmLanguage:183]: This line is commented out, should it stay that way or be removed ?

Comment (alacambre-yseop) [reply syntaxes/yml.tmLanguage.YAML-tmLanguage:183]: Oh. I should remove it. You're right.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,12 @@
 
 ## X.X.X (YYYY-MM-DD)
 
+-   Improved syntax highlighting for:
+    -   text granules
+    -   tags in text granules
+    -   function calls in function calls arguments
```
Comment (vholten-yseop) [inline CHANGELOG.md:8]:
```suggestion
    -   function calls in function call arguments
```

File: syntaxes/yml.tmLanguage.YAML-tmLanguage [hunk 3]
```diff
@@ -200,21 +153,37 @@ repository:
                 match: (\+|-|/|mod|\*)
                 name: keyword.operator.mathematics.yseop
                 comment: 'Some of the operators that can be find in YML.'
```
Comment (vholten-yseop) [inline syntaxes/yml.tmLanguage.YAML-tmLanguage:155]:
```suggestion
                comment: 'Some of the operators that can be found in YML.'
```

**Review by vholten-yseop — COMMENTED** [review]
Looks good, just two typos that I didn't catch the first time.

**Review by vholten-yseop — APPROVED** [review]

**Pull request merged**